scipy/optimize/_elementwise.py

```diff
@@ -226,8 +249,13 @@ def _callback(res):
     else:
         _callback = callback
 
-    res = _chandrupatla(f, xl, xr, args=args, **tolerances,
-                        maxiter=maxiter, callback=_callback)
+    if method in {'chandrupatla', None}:
```

Comment:
We could have a fancier default detection, but for now, `None` is equivalent to `chandrupatla`.